.github/workflows/containers.yml

```diff
@@ -0,0 +1,35 @@
+name: Containers
+on:
+  # Be able to run the job manually when needed
+  workflow_dispatch:
+  # Build every week on Monday 00:00
+  schedule:
+    - cron:  '0 0 * * 1'
```

Comment:
Sunday would be better - nobody works at this time, and if we have some issue, people from TLV
can discover and handle the issue on Sunday.

Reply:
Done